src/model/input.ts

```diff
@@ -101,36 +95,42 @@ class Input {
     return core.getInput('chownFilesTo') || '';
   }
 
-  static get remoteBuildCluster() {
-    return core.getInput('remoteBuildCluster') || '';
-  }
+  static get allowDirtyBuild() {
+    const input = core.getInput('allowDirtyBuild') || false;
 
-  static get awsStackName() {
-    return core.getInput('awsStackName') || '';
+    return input === 'true';
   }
 
-  static get kubeConfig() {
-    return core.getInput('kubeConfig') || '';
+  static get awsStackName() {
+    return core.getInput('awsStackName') || 'game-ci';
   }
 
   static get githubToken() {
     return core.getInput('githubToken') || '';
   }
 
-  static get remoteBuildMemory() {
-    return core.getInput('remoteBuildMemory') || '800M';
+  static get kubeConfig() {
+    return core.getInput('kubeConfig') || '';
   }
 
-  static get remoteBuildCpu() {
-    return core.getInput('remoteBuildCpu') || '0.25';
+  static get kubeVolume() {
+    return core.getInput('kubeVolume') || '';
   }
 
```

Comment:
Probably not hard to solve, but just so you know this could conflict between you and (cc:) @frostebite.

Reply:
Yep, keeping this PR as a draft until after https://github.com/game-ci/unity-builder/pull/310/files is merged, and I'll update afterwards